src/client/components/FormField.less

```diff
@@ -34,6 +34,24 @@
   }
 }
 
+.error {
+ .placeholder;
+ color: red;
+}
+
 .withIcon {
   padding-left: 40px;
-}
\ No newline at end of file
+}
+
+.react-datepicker-wrapper {
```

Comment:
I really wish `react-datepicker` handled this with props. Including specific class names here breaks encapsulation, and means that our stuff will break if someone changes a class in our child dependencies.

Reply:
Yeah I don't like this /at all/ but I at least figured out how to make it fit the container cleaner than the stack overflow answers that recommend editing the react-datepicker code. :p

Again .... maybe a cleaner way if you know what you're doing better than I do and I'm all ears if you have suggestions.